databuilder/models/table_metadata.py

```diff
@@ -117,37 +119,36 @@ def get_description_id(self) -> str:
     def __repr__(self) -> str:
         return 'DescriptionMetadata({!r}, {!r})'.format(self._source, self._text)
 
-    def get_node_dict(self,
-                      node_key: str
-                      ) -> Dict[str, str]:
-        return {
-            NODE_LABEL: self._label,
-            NODE_KEY: node_key,
-            DescriptionMetadata.DESCRIPTION_SOURCE: self._source,
-            DescriptionMetadata.DESCRIPTION_TEXT: self._text or '',
-        }
-
-    def get_relation(self,
-                     start_node: str,
-                     start_key: str,
-                     end_key: str
-                     ) -> Dict[str, str]:
-        return {
-            RELATION_START_LABEL: start_node,
-            RELATION_END_LABEL: self._label,
-            RELATION_START_KEY: start_key,
-            RELATION_END_KEY: end_key,
-            RELATION_TYPE: DescriptionMetadata.DESCRIPTION_RELATION_TYPE,
-            RELATION_REVERSE_TYPE: DescriptionMetadata.INVERSE_DESCRIPTION_RELATION_TYPE
-        }
+    def get_node(self, node_key: str) -> GraphNode:
+        node = GraphNode(
+            key=node_key,
+            label=self._label,
+            attributes={
+                DescriptionMetadata.DESCRIPTION_SOURCE: self._source,
+                DescriptionMetadata.DESCRIPTION_TEXT: self._text
+            }
+        )
+        return node
+
+    def get_relation(self, start_node: str, start_key: Any, end_key: Any) -> GraphRelationship:
+        relationship = GraphRelationship(
+            start_label=start_node,
+            start_key=start_key,
+            end_label=self._label,
+            end_key=end_key,
+            type=DescriptionMetadata.DESCRIPTION_RELATION_TYPE,
+            reverse_type=DescriptionMetadata.INVERSE_DESCRIPTION_RELATION_TYPE,
+            attributes={}
+        )
+        return relationship
 
 
 class ColumnMetadata:
     COLUMN_NODE_LABEL = 'Column'
     COLUMN_KEY_FORMAT = '{db}://{cluster}.{schema}/{tbl}/{col}'
     COLUMN_NAME = 'name'
     COLUMN_TYPE = 'type'
-    COLUMN_ORDER = 'sort_order{}'.format(UNQUOTED_SUFFIX)  # int value needs to be unquoted when publish to neo4j
```

Comment:
hey @AndrewCiambrone , I think we still need to keep this? or i miss the fix somewhere else?

Reply:
nvm, I see the latter part